content/en/docs/collector/management.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Management
+weight: 23
+---
+
+Please be sure to review the following documentation:
+
+*  [Getting Started][otel-collector-getting-started] to understand how to
+   install the OpenTelemetry collector.
+*  [Configuration][otel-collector-configuration] for how to configure the 
+   OpenTelemetry collector, setting up telemetry pipelines.
+
+## Basics
+
+Telemetry collection at scale requires us to come up with a structured approach
+to manage agents. Typical agent management tasks (ordered from most common to
+least common) include:
+
+1. Querying the agent information and configuration. The agent information can
+include its version, operating system related information, or capabilities.
+The configuration of the agent refers to its telemetry collection setup,
+for example, the OpenTelemetry collector [configuration][otel-collector-configuration].
+1. Upgrading and downgrading agents and management of agent-specific packages,
+including the base agent functionality and plugins.
+1. Applying new configurations to agents. This might be required because of
+changes in the environment or due to policy changes.
+1. Health and performance monitoring of the agents, typically CPU and memory usage
+and also agent-specific metrics, for example, the rate of processing or
+backpressure-related information.
+1. Connection management between a control plane and the agent such as
+handling of TLS certificates (revocation and rotation).
+
+Not every use case requires support for all of the above agent management tasks.
+In the context of OpenTelemetry we aim for a dogfooding, so task 
+_4. Health and performance monitoring_ is ideally done using OpenTelemetry.
```

Comment:
Hmm. I don't understand this sentence. Dogfooding in my experience is using the software an org (we) produce as a means to better empathize with end users and fix problems that we all experience. How is that relevant here?

Reply:
Maybe dogfooding is the wrong term. It's a recommendation to use and open standard (OpenTelemetry? :) for this task. Can drop if you think it's not appropriate here.

Reply:
"OpenTelemetry on OpenTelemetry" or something like that?